stdlib/format.mli

```diff
@@ -565,6 +565,14 @@ val pp_get_formatter_out_functions :
    evaluation of these primitives. For instance,
    [print_string] is equal to [pp_print_string std_formatter]. *)
 
+val pp_flush_formatter : formatter -> unit
+(** [pp_flush_formatter fmt] flushes [fmt]'s internal queue, ensuring that all
```

Comment:
The `fmt` here should be `ppf` to be consistent with the rest of the module's documentation.